holoviews/ipython/magics.py

```diff
@@ -364,74 +394,83 @@ def update_options(cls, options, items):
         Switch default options and backend if new backend
         is supplied in items.
         """
-        backend = items.get('backend', '')
+        # Get previous backend
         prev_backend = Store.current_backend
-        renderer = Store.renderers[Store.current_backend]
+        renderer = Store.renderers[prev_backend]
         prev_backend += ':%s' % renderer.mode
 
-        available = backend in Store.renderers.keys()
-        if (not backend) or (not available) or backend == prev_backend:
+        # Get new backend
+        backend = items.get('backend', Store.current_backend)
+        split = backend.split(':')
+        core_backend, mode = split if len(split)==2 else (split[0], 'default')
```

Comment:
It would be nice if we had a word that meant 'backend plus mode' so that 'backend' is the thing without the mode.